File: easy-py/5.zeroFuel.py
# https://www.codewars.com/kata/5861d28f124b35723e00005e/train/python

def zero_fuel(distance_to_pump, mpg, fuel_left):
    if mpg * fuel_left >= distance_to_pump:
        return True
    else:
        return False

result = zero_fuel (100, 50, 1)
print (result)

File: easy-js/5.zeroFuel.js
// https://www.codewars.com/kata/5861d28f124b35723e00005e

// const zeroFuel = (distanceToPump, mpg, fuelLeft) => {
//     return mpg * fuelLeft >= distanceToPump;

// };

const zeroFuel = (distanceToPump, mpg, fuelLeft) =>
  distanceToPump / mpg >= fuelLeft;

console.log(zeroFuel(100, 50, 2));
